code/modules/events/disease_outbreak.dm

```diff
@@ -85,11 +85,13 @@
 	announce_when = ADV_ANNOUNCE_DELAY
 	///The disease type we will be spawning
 	var/datum/disease/virus_type
+	///The preset (classic) or generated (advanced) illness name
+	var/illness_type
```

Comment:
```suggestion
	var/illness_type = ""
```

Do this to "type" it as a string. Could also make it a generic name like `var/illness_type = "Biohazard"` or something, but at least make it string-typed.